pycode/memilio-epidata/memilio/epidata/getCaseData.py

```diff
@@ -333,10 +333,12 @@ def get_case_data(read_data=dd.defaultDict['read_data'],
 
             # cumulative sum over columns defined in dict_files
             if dict_files[file][2] == None:
-                df_local_cs = df_local.cumsum().reset_index(drop=False)
+                df_local_cs = df_local.cumsum()
+                df_local_cs.reset_index(drop=False, inplace=True)
             else:
                 df_local_cs = df_local.groupby(level=[dict_files[file][0].index(
-                    level_index) for level_index in dict_files[file][2]]).cumsum().reset_index()
+                    level_index) for level_index in dict_files[file][2]]).cumsum()
+                df_local_cs.reset_index(drop=False, inplace=True)
```

Comment:
df_local_cs.reset_index(drop=False, inplace=True) should be under the if block bc of code redundancy :)